lib/membrane_rtc_engine/endpoints/hls/video_layout_maker.ex

```diff
@@ -0,0 +1,69 @@
+defmodule Membrane.RTC.Engine.Endpoint.HLS.VideoLayoutMaker do
```

Comment:
I think that this module is very application-specific and we should think about creating an API for creating custom VideoLayoutMaker. This API probably should include options like if we want that specific track should have a border etc. But I am not sure if we should enlarge this PR even more by creating this API. What do you think @mickel8

Reply:
Agree that we should introduce the behavior as soon as possible